### ISSUE TITLE
Awesome Text Skin but change size error

### ISSUE THREAD

**Issue by Mojas84** [grjsmith/Neon-Synthwave-Weather#1]
Hi skin is very nice but small problem how change text size mirror efect on fon not work but are colors of rows......

**Comment by grjsmith** [comment]
Check this section to change all the font sizes at once:

```
[TextStyle]
StringAlign=Left
FontFace=Game Played
;FontFace=Nougat
FontSize=32
Hidden=1
Antialias=1
```

**Comment by Mojas84** [comment]
> Check this section to change all the font sizes at once:

I know but not look same.... This is 22size 
![image](https://github.com/user-attachments/assets/a56d0832-19e7-431c-8983-1d587a95bbd5)
This original 36size 
![image](https://github.com/user-attachments/assets/3d536a49-687d-400e-b6f8-61fe3dd00545)

**Comment by grjsmith** [comment]
Yep that's right, that's expected behaviour. The position of each layer stays fixed when you change the font size. If you go for a larger font size the layers will overlap slightly more, if you go for a smaller font size the layers will separate a little more.

I made the position of each layer static. So you can easily modify the config file to shuffle the layers around. At a 36 pt font Y=15, Y=10, Y=5, Y=0 works well.

If you want to go to 22 pt font try separating the layers by 3 pixels not 5.

**Comment by Mojas84** [comment]
> Yep that's right, that's expected behaviour. The position of each layer stays fixed when you change the font size. If you go for a larger font size the layers will overlap slightly more, if you go for a smaller font size the layers will separate a little more.
> 
> I made the position of each layer static. So you can easily modify the config file to shuffle the layers around. At a 36 pt font Y=15, Y=10, Y=5, Y=0 works well.
> 
> If you want to go to 22 pt font try separating the layers by 3 pixels not 5.

THX very much i use 2pixels on color. 
![image](https://github.com/user-attachments/assets/0d1f64a8-c4c2-4dca-8c51-8b2ba0172fbd)
